hardhat.config.js

```diff
@@ -32,6 +32,29 @@ module.exports = {
       tags: ["local"],
     },
   },
+  external: {
+    contracts: [
+      {
+        artifacts: "node_modules/@keep-network/keep-core/artifacts",
+        // Example if we want to use deployment scripts from external package:
+        // deploy: "node_modules/@keep-network/keep-core/deploy",
+      },
+      {
+        artifacts: "node_modules/@keep-network/tbtc/artifacts",
+      },
+    ],
+    deployments: {
+      development: [
+        "../keep-core/solidity/build/contracts",
+        "../tbtc/solidity/build/contracts",
```

Comment:
Shouldn't we set path to `node-modules`? We link `tbtc` and `keep-core` in `install.sh` (https://github.com/keep-network/coverage-pools/pull/151/commits/7fb710f30ef77dd8a73516afcd28fe9aa46badbd) so I think we should rely on packages.

Reply:
1d90220c3d5ef87beb2be8ca9847c6a3975c8c47